.travis.yml

```diff
@@ -4,7 +4,8 @@ os:
   - linux
   - osx
 julia:
-  - 0.7
+  - '0.7'
+  - '1.0'
```

Comment:
Note that I have to do `- '1.0'` instead of `- 1.0` since the latter would be interpreted as a float by the YAML parser. I changed 0.7 too for symmetry :)